### DISCUSSION TITLE
loading the payload via MCDU

### DISCUSSION THREAD

**Discussion by emsss78** [flybywiresim/aircraft#5548] [Q&A] [answered]
Hello,
Is it possible to load the simbrief payload via the MCDU? I notice if I load it the values don't change in the MSFS fuel page..only the fuel value changes if I load it via the MCDU...I also noticed that after SU5 the payload slider disappeared, so it's kinda difficult adjusting the values without the payload slider.

**Comment by Paytheon** [comment]
Not saying your not having an issue with loading the payload via the MCDU, I can assure you my payload is loading when I press "*Load" via the MCDU with the values I added/configured in SIMBRIEF.

**Reply by emsss78** [reply]
Hmmmmm

**Comment by tracernz** [comment, marked as answer]
> 
> 
> Hello,
> Is it possible to load the simbrief payload via the MCDU? I notice if I load it the values don't change in the MSFS fuel page..only the fuel value changes if I load it via the MCDU...I also noticed that after SU5 the payload slider disappeared, so it's kinda difficult adjusting the values without the payload slider.

The fuel page is broken. If you have loaded your payload and fuel via the MCDU, _do not_ open the MSFS fuel/load page as it will mess things up. Hopefully they will fix it in a future sim update. You can check your FoB and gross weight on the ECAM to confirm your load.

**Reply by emsss78** [reply]
Thanks for the reply Michael....I very much appreciate it. The way I used to do it is to load the fuel via the EFB or MCDU, then manually match the payload in the fuel page with the values from Simbrief by adjusting the weights. So I guess this is totally wrong since you said do not open the fuel page.

**Reply by tracernz** [reply]
> Thanks for the reply Michael....I very much appreciate it. The way I used to do it is to load the fuel via the EFB or MCDU, then manually match the payload in the fuel page with the values from Simbrief by adjusting the weights. So I guess this is totally wrong since you said do not open the fuel page.

Yes. I think something might have broken in one of the sim updates. The MSFS fuel page definitely messes things up now, and isn’t synced to the real state.
